.github/workflows/unitTest.yml

```
@@ -1,4 +1,4 @@
-name: CI
+name: Unit Test
```

```suggestion
name: Unit Tests
```

Also change the file name.